Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -5,10 +5,11 @@ Authors: Zhouhang Zhou, Sébastien Gouëzel, Frédéric Dupuis
 -/
 import Mathlib.Algebra.DirectSum.Module
 import Mathlib.Algebra.Module.LinearMap.Basic
+import Mathlib.Algebra.QuadraticDiscriminant
```

Comment:
Why do we need this import now but we didn't before?

Reply:
I changed the proof of the Cauchy-Schwarz inequality from the previous one (using `definite`) to the current one, so I needed the traditional argument involving discriminant.